### DISCUSSION TITLE
[ FF8 ] Is there a way to change button layout? I get NONE all over the place

### DISCUSSION THREAD

**Discussion by abu-3abed** [julianxhokaxhiu/FFNx#579] [Q&A] [answered]
Is there a way to change button layout? did you put some mechanism to change them in FFNx?

**Comment by julianxhokaxhiu** [comment, marked as answer]
No, that's out of the scope of FFNx. It supports DInput and XInput controllers and you can change them in-game in settings menu.

**Reply by abu-3abed** [reply]
> No, that's out of the scope of FFNx. It supports DInput and XInput controllers and you can change them in-game in settings menu.

oh, I understand. thanks for the reply!

**Comment by abu-3abed** [comment]
didn't mention that I changed keyboard mappings manually in ff8input.cfg as the launcher hinders the user from using special buttons like Shift, del, Enter.. etc.
